catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/search/QueryRunnable.java

```diff
@@ -255,22 +254,13 @@ protected QueryResponse queryCatalog(String sourceId, SearchRequest searchReques
                 }
 
                 LOGGER.debug("Sending query: {}", query);
-                response = searchController.getFramework()
-                        .query(request);
+                ResultIterable results = new ResultIterable(searchController.getFramework(),
+                        request);
+                List<Result> resultsList = new ArrayList<>();
+                results.iterator()
```

Comment:
❗️ As an FYI, the `ResultIterable` iterator will _not_ stop once max query result has been reached but will continue until _all_ results have been retrieved. The `ResultListIterable` could have been used instead, but it doesn't matter since we can't use those here anyways.